Tasks/VSMobileCenterUpload/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vsts-tasks-vsmobilecenterupload",
+  "version": "1.0.0",
+  "description": "VSTS Visual Studio Mobile Center Upload Task",
```

Comment:
User sees the task is in VSTS, so we can drop that and just call it "Visual Studio Mobile Center Upload Task"